spec/fixtures/draupnir.toml

```diff
@@ -0,0 +1,11 @@
+Port = 8443
```

Comment:
I don't think I've ever seen CamelCaps in toml, instead of snake_case. Not super fussed either way, but it would be more consistent as snakecase I think.

Reply:
I was unsure if this would map properly to my config struct - will check. I don't think toml specifies a preference - [Cargo uses kebab-case](https://doc.rust-lang.org/cargo/reference/manifest.html) for its config.

Reply:
I love kebabs, and am ambivalent provided we drop the camelcase as that feels somehow wrong to me in toml.

Reply:
My point is that there's no set style - Go itself advocates camelCase, and if you check the `.draupnir` file that the cli uses and generates, it's also in camelCase.

Reply:
Sorry, capitalised CamelCase. Can we just remove the leading uppercase character?